ocs_ci/ocs/resources/mcg.py

```diff
@@ -907,91 +913,57 @@ def exec_mcg_cmd(self, cmd, namespace=None, use_yes=False, **kwargs):
     )
     def retrieve_noobaa_cli_binary(self):
         """
-        Copy the NooBaa CLI binary from the operator pod
-        if it wasn't found locally, or if the hashes between
-        the two don't match.
+        Download the MCG CLI from quay based on OS type
 
         Raises:
-            NoobaaCliChecksumFailedException: If checksum doesn't match.
-            AssertionError: In the case CLI binary doesn't exist.
+            UnsupportedOSType: OS type is not supported.
 
         """
+        ocs_version = get_ocs_build_number()
+        pull_secret_path = os.path.join(constants.DATA_DIR, "pull-secret")
+        mcg_cli_image_location = os.path.join(
+            config.ENV_DATA["cluster_path"], f"mcg-cli-images-{config.RUN['run_id']}"
```

Comment:
Must we keep track of the images that were used in previous runs? If we replace `{config.RUN['run_id']}` with `{ocs_version}` here it would allow us to check if it already exists before downloading it, and save a lot of time when running OCS-CI locally.